src/TemplateUI/Controls/Slider/Slider.cs

```diff
@@ -170,14 +161,20 @@ void OnThumbPanUpdated(object sender, PanUpdatedEventArgs e)
 
                     if (Device.RuntimePlatform == Device.Android)
                         totalX += _thumb.TranslationX;
-
-                    var position = totalX < 0 ? 0 : totalX > _trackBackground.Width - _thumb.Width ? _trackBackground.Width - _thumb.Width : totalX;
-                    Value = position * Maximum / _trackBackground.Width;
+                    
```

Comment:
I will review the calculations, when moving to the right there is at the beginning a little jump to the left. 
![thumbsize](https://user-images.githubusercontent.com/6755973/108623715-b480c900-7440-11eb-9ca9-4a928ab5733b.gif)